addons/binding/pom.xml

```diff
@@ -67,6 +67,7 @@
     <module>org.openhab.binding.squeezebox</module>
     <module>org.openhab.binding.systeminfo</module>
     <module>org.openhab.binding.systeminfo.test</module>
+    <module>org.openhab.binding.tankerkoenig</module>
```

Comment:
All the binding should be more or less alphabetical in there, they start their name with 'openhab-binding-'